### PR TITLE
New package: net-p2p/dogecoin-qt

### PR DESCRIPTION
**Pull request by victorsk2019**
Hi everyone,

This is a new PR for net-p2p/dogecoin-qt for this repository.

Thank you.

### CODE REVIEW

File: net-p2p/dogecoin-qt/dogecoin-qt-1.14.6.ebuild [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+
+src_configure() {
+	chmod 755 ./autogen.sh
+	./autogen.sh || die "autogen failed"
+	local my_econf=(
+		--enable-cxx
+		$(use_with wallet incompatible-bdb)
+		--bindir="${DOGEDIR}/bin"
+		CPPFLAGS="-I/usr/include/db${DB_VER}"
+		CFLAGS="-I/usr/include/db${DB_VER}"
```
Comment (N-R-K) [inline net-p2p/dogecoin-qt/dogecoin-qt-1.14.6.ebuild:59]: ebuilds need to respect user's CFLAGS, and why is this duplicated in CPPFLAGS also?

Comment (victorsk2019) [reply net-p2p/dogecoin-qt/dogecoin-qt-1.14.6.ebuild:59]: > why is this duplicated in CPPFLAGS also?

I used it as a "safety net" to ensure the right db header files are used and software builds correctly.

**Comment by victorsk2019** [comment]
Note: I am also working on the "prune" USE flag, so users would have an option to download a more trimmed version of the blockchain.

**Comment by N-R-K** [comment]
Btw, you should apply for access to guru directly: https://wiki.gentoo.org/wiki/Project:GURU/Information_for_Contributors#Prior_setup - this way you can push directly to the `dev` branch. That's the preferred way to contribute to guru - with github pull request, you have no guarantee if someone is availble to merge your PR or not.

(In this case, I'm not interested in this package and thus don't want to go through the trouble of testing it to merge it).

**Comment by victorsk2019** [comment]
> Btw, you should apply for access to guru directly: https://wiki.gentoo.org/wiki/Project:GURU/Information_for_Contributors#Prior_setup - this way you can push directly to the `dev` branch. That's the preferred way to contribute to guru - with github pull request, you have no guarantee if someone is availble to merge your PR or not.

Thanks for the info.  I'll definitely check it out.

P.S. This package runs like a beast in Gentoo 💪. I like it!

**Comment by victorsk2019** [comment]
Switched to 'dev' branch as advised.

**Pull request closed without merging**